src/module/chat-message/document.ts

```diff
@@ -199,11 +199,9 @@ class ChatMessagePF2e extends ChatMessage {
         if (actor && header && this.isContentVisible) {
             const token = this.token ?? actor.prototypeToken;
             const useToken =
-                actor.prototypeToken?.texture?.src &&
-                ImageHelper.hasImageExtension(token.texture.src) &&
-                !isDefaultTokenImage(token);
+                token?.texture?.src && ImageHelper.hasImageExtension(token.texture.src) && !isDefaultTokenImage(token);
```

Comment:
Is this nullable? The type may just need to be tighter if it's a `string`.

Reply:
Its not, but it seems if you try feeding it a blank string foundry turns it into mystery man.

Reply:
_Can_ it be blank?